apps/laboratory/package.json

```diff
@@ -11,11 +11,11 @@
     "playwright:test": "npx playwright test",
     "playwright:test:wallet": "npx playwright test --grep 'connect-qr.spec.ts|wallet.spec.ts'",
     "playwright:test:siwe": "npx playwright test --grep siwe.spec.ts",
-    "playwright:test:canary": "npx playwright test --grep canary.spec.ts --project='Desktop Chrome/wagmi'",
-    "playwright:debug": "npx playwright test --debug",
-    "playwright:debug:wallet": "npx playwright test --debug 'connect-qr.spec.ts|wallet.spec.ts'",
-    "playwright:debug:siwe": "npx playwright test --debug --grep siwe.spec.ts",
-    "playwright:debug:canary": "npx playwright test --debug --grep canary.spec.ts --project='Desktop Chrome/wagmi'"
+    "playwright:test:canary": "npx playwright test --retries=0 --grep canary.spec.ts --project='Desktop Chrome/wagmi'",
+    "playwright:debug": "npm run playwright:test -- --debug",
+    "playwright:debug:wallet": "npm run playwright:test:wallet -- --debug",
+    "playwright:debug:siwe": "npm run playwright:test:siwe -- --debug",
+    "playwright:debug:canary": "npm run playwright:test:canary -- --debug"
```

Comment:
do you not need `--project='Desktop Chrome/wagmi'` anymore?

Reply:
We don't here, because now instead we are calling the other NPM script which does have it